.github/workflows/ci.yml

```diff
@@ -40,10 +40,11 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.9]
+        python-version: ["3.12"]
```

Comment:
Why this change, aren't we removing 3.8?

Reply:
- 3.12 is the current stable version, so that's the one we should test across platforms
- 3.8 is still supported by upstream

Reply:
![image](https://github.com/quickwit-oss/tantivy-py/assets/480395/b37a27d8-7ed3-42bf-a1ec-90116b3314b5)